src/fsharp/FSharp.Core/async.fsi

```diff
@@ -114,8 +157,18 @@ namespace Microsoft.FSharp.Control
         /// <returns>A computation that returns a choice of type T or exception.</returns>
         ///
         /// <category index="3">Cancellation and Exceptions</category>
-        /// 
-        /// <example-tbd></example-tbd>
+        ///
+        /// <example id="catch-example-1">
+        /// <code lang="fsharp">
+        /// async { return someRiskyBusiness() }
```

Comment:
It's best to define the function and make the example complete.  If you like define it to look at System.DateTime.Now.DayOfWeek and fail on Monday